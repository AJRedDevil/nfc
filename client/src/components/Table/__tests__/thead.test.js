// npm packages
import React from 'react';
import renderer from 'react-test-renderer';
import {range} from 'lodash';

// our packages
import THead, {TableHeader} from '../thead';

describe('# THead', () => {
  test('TableHeader', () => {
    const tree = renderer.create(<TableHeader text="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('THead', () => {
    const data = range(1, 5).map(i => ({text: `Test${i}`}));
    const tree = renderer.create(<THead data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
