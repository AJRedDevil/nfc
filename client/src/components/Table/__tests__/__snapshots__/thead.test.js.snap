// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# THead THead 1`] = `
<thead>
  <tr>
    <th>
      Test1
    </th>
    <th>
      Test2
    </th>
    <th>
      Test3
    </th>
    <th>
      Test4
    </th>
  </tr>
</thead>
`;

exports[`# THead TableHeader 1`] = `
<th>
  test
</th>
`;
